src/app/campaign/store/reducers/flight.reducer.ts

```diff
@@ -98,7 +98,10 @@ export function reducer(state = initialState, action: CampaignActions): State {
           id: flight.id,
           changes: {
             localFlight,
-            changed,
```

Comment:
Some of the flight form update events had the form.dirty: false, so I decided that with form updates, changed for each flight should be false until true and then remain true until "Save".

Reply:
+1 makes sense.  Since we don't allow "discard" anyways ... may as well keep it marked changed until save.